### PR TITLE
Create_CRUD

### PR DESCRIPTION
**Pull request by mbrown78**
Added CRUD, controllers Dashboard and Games. Created basic methods new, create and show for games controller. Added bootstrap and nav bar. Added test for game controller create action and show. Having trouble passing the show#action test.

### CODE REVIEW

File: app/controllers/dashboard_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class DashboardController < ApplicationController
```
Comment (lasley) [inline app/controllers/dashboard_controller.rb]: This is out of scope for game CRUD

File: app/helpers/dashboard_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module DashboardHelper
```
Comment (lasley) [inline app/helpers/dashboard_helper.rb]: Out of scope for Game CRUD

File: app/controllers/dashboard_controller.rb (hunk 1 above)
Comment (lasley) [inline app/controllers/dashboard_controller.rb]: Why a separate dashboard? What will go here?

File: app/views/games/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<h1> a simple form create new game </h1>
```
Comment (lasley) [inline app/views/games/new.html.erb]: Is this the title you want on this page?

File: app/views/games/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+<h1>show new game here</h1>
```
Comment (lasley) [inline app/views/games/show.html.erb]: Is this the title you want on this page?

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -18,5 +45,3 @@
 
 <%= yield %>
 
-</body>
-</html>
```
Comment (lasley) [inline app/views/layouts/application.html.erb]: Why remove these tags? HTML RFC requires them

File: spec/controllers/games_controller_spec.rb [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    it "should successfully show the page if the game is found" do
+       game = FactoryGirl.create(:game)
+       get :show, id: game.id
+       expect(response).to have_http_status(:success)
+    end
+    it "should return a 404 error if the game is not found" do 
+       get :show, id: 'TACOCAT'
+       expect(response).to have_http_status(:not_found)
+    end  
+  end
```
Comment (lasley) [inline spec/controllers/games_controller_spec.rb]: What about `game#new action`?

**Comment by lasley** [comment]
@mbrown78 Comments inline. Please also fix the build and resolve merge conflicts

**Comment by mbrown78** [comment]
we are disregarding this pull request and using the "New list game seats " as the updated version
